### DISCUSSION TITLE
Geometric Parameter for Scaling LFP Battery Design

### DISCUSSION THREAD

**Discussion by jitendraec** [pybamm-team/PyBaMM#4446] [Q&A] [answered]
I am working on scaling up an LFP battery cell design, using the Prada2013 parameters as a reference. The original cell has a capacity of 2.3 Ah, and I am aiming to develop a 72 Ah cell. I applied a scale-up factor of 72/2.3 to base my calculations.

My question is: as I scale up the nominal capacity, will the geometric parameters like electrode and separator thickness change? Or will only the length and width of the electrode need adjustment while maintaining same the thickness?

Any clarification on how these parameters should evolve during scaling would be greatly appreciated.

Thank you!

**Comment by rtimms** [comment, marked as answer]
The parameter "Nominal cell capacity [A.h]" is simply used to convert between current and C-rate and does not affect the actual capacity of the cell. The easiest way to increase the _actual_ capacity is to change the "Electrode height [m]" and "Electrode width [m]" parameters. The product of these gives the electrode cross-sectional area of the electrode pair. You can also change "Number of electrodes connected in parallel to make a cell".

**Reply by jitendraec** [reply]
Thank You!

**Reply by Notslytherin1** [reply]
@jitendraec, did you manage to solve this problem in the end? I have done the same thing, suggested by multiplying the width by the number of layers the cell has, which indeed helped me achieve a higher capacity value. However, I noticed that the voltage curve shifts up, or sometimes the amplitude of the voltage gets reduced after just 3 cycles. did you face something similar?
